app/src/forms/admin/routes.js

```diff
@@ -65,18 +390,149 @@ routes.get('/users/:userId', async (req, res, next) => {
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/responses/responseBody/AdminGetProactiveHelpObject'
+ *    '403':
+ *      $ref: '#/components/responses/Error/AccessDenied'
+ *    '5XX':
+ *      $ref: '#/components/responses/Error/UnExpected'
+ *    '422':
+ *      $ref: '#/components/responses/Error/UnprocessableEntityDB'
+ */
 routes.post('/formcomponents/proactivehelp/object', async (req, res, next) => {
   await controller.createFormComponentsProactiveHelp(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/formcomponents/proactivehelp/{publishStatus}/{componentId}:
+ *  put:
+ *   tags:
+ *    - Admin
+ *   summary: update form.io component proactive help publish status
+ *   description: This endpoint will update form.io component proactive help publish status.
+ *   security:
+ *    - openId: []
+ *   parameters:
+ *      - in: path
+ *        name: publishStatus
+ *        schema:
+ *          type: boolean
+ *        description: Publish status of the Firm.io component.
+ *        required: true
+ *        example: true
+ *      - in: path
+ *        name: componentId
+ *        schema:
+ *          type: string
+ *        description: Id of the Form.io component.
+ *        required: true
+ *        example: true
+ *   responses:
+ *    '200':
+ *      description: Success
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/responses/responseBody/AdminProactiveHelpPublishStatus'
+ *    '403':
+ *      $ref: '#/components/responses/Error/AccessDenied'
+ *    '5XX':
+ *      $ref: '#/components/responses/Error/UnExpected'
+ *    '422':
+ *      $ref: '#/components/responses/Error/UnprocessableEntityDB'
+ *    '404':
+ *        $ref: '#/components/responses/Error/ResourceNotFound'
+ */
 routes.put('/formcomponents/proactivehelp/:publishStatus/:componentId', async (req, res, next) => {
   await controller.updateFormComponentsProactiveHelp(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/formcomponents/proactivehelp/imageUrl/{componentId}:
+ *  get:
+ *   tags:
+ *    - Admin
+ *   summary: Get the image of the form.io component proactive help
+ *   description: This endpoint will get the image of the form.io component proactive help.
+ *   security:
+ *    - openId: []
+ *   parameters:
+ *    - in: path
+ *      name: componentId
+ *      schema:
+ *        type: string
+ *      description: Id of the Form.io component.
+ *      required: true
+ *      example: true
+ *   responses:
+ *    '200':
+ *      description: Success
+ *      content:
+ *        image/png:
+ *          schema:
+ *            type: object
+ *            properties:
+ *              url:
+ *                type: string
+ *                format: binary
+ *    '403':
+ *      $ref: '#/components/responses/Error/AccessDenied'
+ *    '5XX':
+ *      $ref: '#/components/responses/Error/UnExpected'
+ *    '422':
+ *      $ref: '#/components/responses/Error/UnprocessableEntityDB'
```

Comment:
Is this an actual response?

Reply:
Yes

Reply:
Yikes! We definitely have some cleanup todo - thanks!